nlmod/gwf/output.py

```diff
@@ -129,7 +221,7 @@ def _get_hds(ds=None, gwf=None, fname_hds=None):
     return headobj
 
 
-def _get_cbc(ds=None, gwf=None, fname_cbc=None):
+def _get_cbf(ds=None, gwf=None, fname_cbc=None):
```

Comment:
I think it should be cbc? If not the argument `fname_cbc` should be changed as well?